alacritty_terminal/Cargo.toml

```diff
@@ -50,3 +44,4 @@ windows-sys = { version = "0.48.0", features = [
 
 [dev-dependencies]
 serde_json = "1.0.0"
+alacritty_terminal = { path = ".", features = ["serde"] }
```

Comment:
Huh?

Reply:
I need serde feature in tests, do you know other syntax to enable it?

Reply:
Could just make it a default feature.